linters/nelua.lua

```diff
@@ -0,0 +1,36 @@
+-- Nelua plugin for lint+
+
+--- CONFIG ---
+
+-- config.lint.nelua_mode: "analyze" | "lint"
+--   changes the linting mode, "analyze" (default) does a complete checking,
+--   while "lint" only checks for syntax errors.
+
+--- IMPLEMENTATION ---
+
+local core = require 'core'
+local lintplus = require 'plugins.lintplus'
+
+local mode = lintplus.config.nelua_mode or "analyze"
+
+if mode ~= "analyze" and mode ~= "lint" then
+  core.error("lint+/nelua: invalid nelua_mode '%s'. Available modes: 'analyze', 'lint'", mode)
+end
+
+local command = lintplus.command {
+  'nelua',
+  '--no-color',
+  '--'..mode,
+  lintplus.filename
+}
+
+lintplus.add 'nelua' {
+  filename = '%.nelua$',
+  procedure = {
+    command = command,
+    interpreter = lintplus.interpreter {
+      error = "(.-):(%d+):(%d+):.-error: (.+)"
```

Comment:
Does nelua never emit any warnings? Only errors?

Reply:
Yes, it's only errors. (There is a warning system, but it's not for compile-time).